addons/website/static/src/builder/plugins/form/form_option_plugin.js

```diff
@@ -268,6 +267,9 @@ export class FormOptionPlugin extends Plugin {
             }));
         } else if (field.relation && field.relation !== "ir.attachment") {
             const fieldNames = field.fieldName ? [field.fieldName] : ["display_name"];
+            if (field.name === "state_id") {
+                fieldNames.push("country_id");
+            }
```

Comment:
What happens if the model has a "state_id" field but don t have a "country_id" field ?